src/components/Message/MessageSimple/ReactionList.tsx

```diff
@@ -87,18 +84,15 @@ export type ReactionListPropsWithContext<
 > = Pick<
   MessageContextValue<At, Ch, Co, Ev, Me, Re, Us>,
   'alignment' | 'onLongPress' | 'reactions' | 'showMessageOverlay'
-> &
-  Pick<
```

Comment:
this fixes the TS infinite error as the context contains both this component and supportedReactions